### ISSUE TITLE
go.Scatter bug on filled area rendering

### ISSUE THREAD

**Issue by jjnurminen** [plotly/plotly.py#1008]
The plotly examples suggest doing filled curves by `go.Scatter`. I'm trying to fill a rectangular box. The following code works fine:

```
import plotly
import plotly.graph_objs as go

yval = 1e2
ntrace = go.Scatter(x=[1, 10, 10, 1], y=[yval, yval, -yval, -yval], fill='toself')
fig = go.Figure(data=[ntrace])
fig['layout']['yaxis'].update(range=[-.5, .5])
plotly.offline.plot(fig)
```
Result as expected:
![newplot](https://user-images.githubusercontent.com/8491321/39985732-34110502-5768-11e8-9890-5127b56477af.png)

However after changing to `yval = 1e3`, I get:

![newplot 2](https://user-images.githubusercontent.com/8491321/39985894-b392db0c-5768-11e8-84ef-36271f12c183.png)

In other words, zooming the plot with the bigger rectangle seems to screw up its dimensions.

**Comment by jonmmease** [comment]
Hi @jjnurminen , I just tried this out with version 3.2.1 and I'm not seeing the issue in your screenshots above. If you're still interested, could you give it another try with version 3.2+?  Thanks!

**Comment by jjnurminen** [comment]
Thanks @jonmmease 

I just updated to 3.2.1. If I change to `yval = 1e3` in the code above, I get the following:

![newplot](https://user-images.githubusercontent.com/8491321/45962699-22cfc000-c02a-11e8-824c-85aa074dfb7f.png)

So it's screwed up in slightly different way.

**Comment by jonmmease** [comment]
Moving this issue to the plotly.js tracker at https://github.com/plotly/plotly.js/issues/3147.  Thanks!
